### DISCUSSION TITLE
When will version 0.4 be available?

### DISCUSSION THREAD

**Discussion by bluehysteric** [ivangabriele/clamav-desktop#87] [Questions & Answers] [answered]
When will version 0.4 be available?

**Comment by ivangabriele** [comment, marked as answer]
Hello @bluehysteric!

I'm quite irregular on this project but I started again working on it locally this week. Hopefully I expect to finally reach `v0.4.0` by the beginning of November. After that everything will get a lot easier because the new Rust + Tauri base will have been entirely implemented.

**UPDATE 2004-06-08**

I won't give any deadline anymore since it's quite impossible for me to predict my workload. However I'm still actively working on it.

**Reply by bluehysteric** [reply]
Excellent, we need 0.4 so badly!

**Reply by DarkPhoinix** [reply]
Yes!

**Comment by Neustradamus** [comment]
Linked to:
- https://github.com/ivangabriele/clamav-desktop/issues/235
- https://github.com/ivangabriele/clamav-desktop/discussions/237
- https://github.com/ivangabriele/clamav-desktop/issues/216
- https://github.com/ivangabriele/clamav-desktop/discussions/239
- https://github.com/ivangabriele/clamav-desktop/issues/211
- https://github.com/ivangabriele/clamav-desktop/discussions/238
- https://github.com/ivangabriele/clamav-desktop/issues/187
- https://github.com/ivangabriele/clamav-desktop/discussions/190
- https://github.com/ivangabriele/clamav-desktop/issues/180
- https://github.com/ivangabriele/clamav-desktop/discussions/188
- https://github.com/ivangabriele/clamav-desktop/issues/127
- https://github.com/ivangabriele/clamav-desktop/discussions/131
- https://github.com/ivangabriele/clamav-desktop/issues/118
- https://github.com/ivangabriele/clamav-desktop/discussions/130
- https://github.com/ivangabriele/clamav-desktop/issues/85
- https://github.com/ivangabriele/clamav-desktop/discussions/87
